persistence/src/commonMain/kotlin/com/wire/kalium/persistence/config/UserConfigStorage.kt

```diff
@@ -458,7 +458,7 @@ class UserConfigStorageImpl(
             E2EI_NOTIFICATION_TIME,
             timeStamp
         ).also {
-            e2EIFlow.tryEmit(Unit)
+            e2EINotificationFlow.tryEmit(Unit)
```

Comment:
this doesn't relate to the original task, i just found some copy-past mistake (which causes unknown yet issue) and fixed it :)